awx/ui/src/screens/InstanceGroup/Instances/InstanceList.js

```diff
@@ -56,6 +58,12 @@ function InstanceList({ instanceGroup }) {
         InstanceGroupsAPI.readInstances(instanceGroupId, params),
         InstanceGroupsAPI.readInstanceOptions(instanceGroupId),
       ]);
+      response.data.results.forEach((i) => {
```

Comment:
I'm a bit worried that this could possibly reset `pendingHealthCheck` repeatedly if the list has lots of items that are in both states.  Could we use a `.some` here?